mathjax3-ts/output/common/Wrapper.ts

```diff
@@ -551,13 +551,16 @@ AbstractWrapper<MmlNode, CommonWrapper<J, W, C>> {
             indentalign = indentalignfirst;
         }
         if (indentalign === 'auto') {
-            indentalign = 'center';
+            indentalign = this.jax.options.displayAlign;
         }
         if (indentshiftfirst !== 'indentshift') {
             indentshift = indentshiftfirst;
         }
         if (indentshift === 'auto') {
-            indentshift = '0';
+            indentshift = this.jax.options.displayIndent;
+            if (indentalign === 'right' && !indentshift.match(/^\s*0[a-z]*\s*$/)) {
+                indentshift = ('-' + indentshift.replace(/^\s+/, '')).replace(/^--/, '');
```

Comment:
Couldn't that simply be done with a `trim()` or is the whitespace on the right important?

Reply:
Yes, that can be used.  I'll fix it.